malaria24/settings/base.py

```diff
@@ -166,7 +172,6 @@
 # https://docs.djangoproject.com/en/1.7/ref/settings/#databases
 
 # SQLite (simplest install)
-import dj_database_url
```

Comment:
Why are we removing this?

Reply:
It was added in the middle of the code, and thus created flake8 errors, so I moved it to the top/beginning of the file

Reply:
Oops I missed that.